src/main/java/racing/Car.java

```diff
@@ -0,0 +1,13 @@
+package racing;
+
+public class Car {
+    private int moves = 0;
+
+    public void move() {
+        moves += 1;
```

Comment:
증감 연산자를 활용하면 `1`과 같은 지역 변수를 사용하지 않아도 될 것 같아요.
```suggestion
        moves++;
```